### DISCUSSION TITLE
Can not load system configuration on the crazyfile

### DISCUSSION THREAD

**Discussion by j-roman-tic** [bitcraze/discussions#1403] [Q&A]
Hi,
Currently working with the Light House (LH) system. I tried to load a .yaml file with a previous configuration on a crzayflie drone but something went wrong and now the led M4 is blinking all the time as well as the led on the Crazyradio Antena...as if it was sending something to the drone.
I tried changing the antena and even tried other drones with different adresses but every time I connect to the CF client the M4 led starts blinking rapidly and the system configuration never finishes.
Is that normal ? If not, how can I 'stop' the uploading and how can I do it properly ?
If anyone knows how to help me, I will be very grateful :)

**Comment by gemenerik** [comment]
A fast blinking M4 when connected is expected behavior. Loading a lighthouse configuration should not affect this. Can you elaborate on what went wrong when trying to load the previous lighthouse configuration? 

Do you mean the initialization of the drone does not finish? Perhaps you can open the console tab in the client and copy what it prints upon initialization to this discussion?
